### DISCUSSION TITLE
saving panel location on screen

### DISCUSSION THREAD

**Discussion by matluuk** [GrandOrgue/grandorgue#1534] [Help] [answered]
I use two touch screens, and I need to use multiple panels on the screens. 

Is it possible to save panel location on screen so that panels don't open in the middle of the scrren when opening an organ?

Now I have to move panels every time I open organ, which is not optimal.

**Comment by midimusic** [comment]
Yes it is possible but it has to be implemented in the code. Screen coordinates of the panel have to be saved when it is cliosed and restored when it is re-opened. I have just tested the Windows GO and it appears that this feature has not been implemented.

csw900

**Comment by oleg68** [comment, marked as answer]
@matluuk 

1. Install the latest version of GO from Github
2. File->Save

**Comment by matluuk** [comment]
Thank you. I updated 3.08 to latest and it now works.
